src/databricks/labs/remorph/reconcile/query_builder/expression_generator.py

```diff
@@ -77,23 +79,28 @@ def anonymous(expr: exp.Expression, func: str) -> exp.Anonymous | exp.Expression
     To achieve the same,we can use the function as below:
     eg:
         >>> expr = parse_one("select col1 from dual")
-        >>> transformed_expr=anonymous(expr,"unix_timestamp")
+        >>> transformed_expr=anonymous(expr,"unix_timestamp({})")
         >>> print(transformed_expr)
         'SELECT UNIX_TIMESTAMP(col1) FROM DUAL'
 
     """
+    level = 0 if isinstance(expr, exp.Column) else 1
```

Comment:
why do we needs this?

Reply:
We need this to handle complex statements like (COALESCE(source.s_acctbal, 0) - COALESCE(databricks.s_acctbal, 0)). Also it was not handling table name and column name earlier